src/app/pages/apps/components/available-apps/available-apps-header/available-apps-header.component.spec.ts

```diff
@@ -62,8 +63,8 @@ describe('AvailableAppsHeaderComponent', () => {
     await filtersButton.click();
 
     searchInput = await loader.getHarness(MatInputHarness.with({ placeholder: 'Search' }));
-    catalogsSelect = (await loader.getAllHarnesses(MatSelectHarness))[0];
-    sortSelect = (await loader.getAllHarnesses(MatSelectHarness))[1];
+    catalogsItems = spectator.queryAll('.catalogs .item');
+    sortItems = spectator.queryAll('.sort .item');
```

Comment:
This is fine, but it would be easier to work with the component in tests if you were to create a harness for `filter-select-list`.